forc/src/ops/forc_init.rs

```diff
@@ -106,82 +74,92 @@ pub fn init(command: InitCommand) -> Result<()> {
         (false, false, false, true) => Library,
         _ => anyhow::bail!(
             "Multiple types detected, please specify only one program type: \
-                \n Possible Types:\n - contract\n - script\n - predicate\n - library"
+        \n Possible Types:\n - contract\n - script\n - predicate\n - library"
         ),
     };
 
-    init_new_project(project_name, program_type)
-}
-
-pub(crate) fn init_new_project(project_name: String, program_type: ProgramType) -> Result<()> {
-    let neat_name: String = project_name.split('/').last().unwrap().to_string();
-
     // Make a new directory for the project
-    fs::create_dir_all(Path::new(&project_name).join("src"))?;
+    fs::create_dir_all(Path::new(&project_dir).join("src"))?;
+
+    if command.verbose {
+        println_light_blue(&format!(
+            "\nUsing project directory at {}",
+            canonicalize(&project_dir)?
+        ))
+    }
```

Comment:
Perhaps we should move this output earlier, right after we retrieve `project_dir` and check that it's valid?